tests/protocol/test_wire.py

```diff
@@ -61,6 +60,16 @@ def get_event(message, duration=30000, evt_type="", is_internal=False, is_succes
     return event
 
 
+def setup_wireclient_ext_conf(wire_protocol_client, artifacts_profile_blob=None,
```

Comment:
Helper function to create and set an Env_config object for the tests (made it global as its being used in multiple test classes)

Reply:
nit: 'set up' is a verb needed in this scenario, 'setup' is a noun.

Reply:
Thanks for catching this, changed the name